src/components/Cards/CardBody.js

```diff
@@ -0,0 +1,20 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CardBody = ({ children, className, ...props }) => {
+  const classes = ClassNames('p', 'card-pf-body', className);
+
+  return (
+    <p className={classes} {...props}>
```

Comment:
should render a `div` instead i believe... the paragraph can follow in children...